notification/docs/IntegrationGuide.md

```diff
@@ -58,6 +58,14 @@ Replace all `XXX` values and execute:
 docker run -e CTP_PROJECT_KEY=XXX -e CTP_CLIENT_ID=XXX -e CTP_CLIENT_SECRET=XXX ctp-adyen-integration-notification:XXX
 ```
 
+### Deployment using AWS Lambda
+
+##### Deploying the Lambda Code
+For deployment to lambda, zip the notification folder and specify the src/lambda.handler as the entry point for the function
```

Comment:
```suggestion
For deployment to lambda, zip the notification folder and specify `src/lambda.js` as the entry point for the function
```

Reply:
Have brought this in line with the changes from the extension PR